server/views/pages/overview/viewArchivedGoals.njk

```diff
@@ -31,26 +31,30 @@
     <div class="govuk-grid-column-two-thirds app-u-print-full-width">
       <section data-qa="view-archived-goals-list">
         {% if not goals.problemRetrievingData %}
-          {% for goal in goals.goals | sort(attribute = 'updatedAt', reverse = true) %}
-            {{ goalSummaryCardV2({
-              goal: goal,
-              attributes: {
-                'data-qa': 'goal-summary-card'
-              },
-              classes: 'archived-goal-summary-card',
-              lastUpdatedLabel: 'Archived on',
-              actions: [
-                {
-                  title: 'Reactivate<span class="govuk-visually-hidden"> this goal</span>',
-                  href: '/plan/' + prisonerSummary.prisonNumber + '/goals/' + goal.goalReference + '/unarchive',
-                  attributes: {
-                  'data-qa': 'goal-'+ loop.index + '-unarchive-button'
+          {% if goals.goals | length > 0 %}
+            {% for goal in goals.goals | sort(attribute = 'updatedAt', reverse = true) %}
+              {{ goalSummaryCardV2({
+                goal: goal,
+                attributes: {
+                  'data-qa': 'goal-summary-card'
                 },
-                  'render-if': hasEditAuthority
-                }
-              ]
-            }) }}
-          {% endfor %}
+                classes: 'archived-goal-summary-card',
+                lastUpdatedLabel: 'Archived on',
+                actions: [
+                  {
+                    title: 'Reactivate<span class="govuk-visually-hidden"> this goal</span>',
+                    href: '/plan/' + prisonerSummary.prisonNumber + '/goals/' + goal.goalReference + '/unarchive',
+                    attributes: {
+                      'data-qa': 'goal-'+ loop.index + '-unarchive-button'
+                    },
+                    'render-if': hasEditAuthority
+                  }
+                ]
+              }) }}
+            {% endfor %}
+          {% else %}
+           <p data-qa="no-archived-goals-message">{{ prisonerSummary.firstName + ' ' + prisonerSummary.lastName }} has no archived goals</p>
```

Comment:
When the page renders, does this paragraph tag display with the correct govuk styles/font?
I never know whether the styles inherit and cascade etc (eg: I dont know if classes used on parent elements such as `govuk-grid-column-two-thirds` set the font etc for `p` elements etc)
So often times I explicitly set the `govuk-body` class on the `p` tag:

```
<p class="govuk-body" data-qa=......
``` 

I don't know if we need to do that or not; like I said its mostly based on my lack of understanding about how the govuk classes and styles are setup. 
Would probably be worth checking it displays OK though 👍

Reply:
It did display correctly so I forgot to add the class!